### DISCUSSION TITLE
Support for other frameworks?

### DISCUSSION THREAD

**Discussion by Dindaleon** [skeletonlabs/skeleton#1393] [General]
I would love to use skeleton for Qwik! Is there any chance it will be supported any time soon?

**Comment by endigo9740** [comment]
@Dindaleon I'd recommend checking out or recently posted Roadmap for Skeleton:
- https://github.com/skeletonlabs/skeleton/discussions/1367

Near the bottom we cover some of our long term goals for the project. Other frameworks are a consideration, but these will be will be a long term effort with a lot of heavy prerequisites first. That said, we are looking for experts in other frameworks that can help with this effort. If you're interested, let me know.

But I strongly recommend you temper expectations here. It took over a year to get Skeleton to where it is now with a single framework.

**Comment by Dindaleon** [comment]
I just heard about skeleton ui a few days ago and fell in love with it. However, I am about to start a project using the relative new qwik framework which is very promising and so easy to work with. I don't know if I will be able to help porting skeleton ui to qwik, but I will do my research and let you know in case I decide to help.
